webpack/assets/javascripts/react_app/redux/API/APIHelpers.js

```diff
@@ -0,0 +1,26 @@
+import { API_OPERATIONS } from './APIConstants';
+import { API } from './';
+
+export const getApiMethodByActionType = type =>
+  Object.keys(API_OPERATIONS)
+    .find(apiKey => API_OPERATIONS[apiKey] === type)
+    .toLowerCase();
+
+export const getApiResponse = async ({ type, url, headers, params }) => {
+  const method = getApiMethodByActionType(type);
+  let response = {};
+  switch (method) {
+    case 'get':
+      response = API[method](url, headers, params);
+      break;
+    case 'delete':
+      response = API[method](url, headers);
```

Comment:
how about returning it instead of storing it? it saves the need of a `break`
```js
return API[method](url, headers, params);
```
BTW, is there an option to use a general axios object instead of a switch case?
```js
axios({
  method,
  url,
  data: params
  headers,
  // etc
});

Reply:
that's a really good idea,
I was also investigated it in this PR,
though there are some tests mocks to the 'API.js' file that fails:

![Selection_256](https://user-images.githubusercontent.com/26363699/72434801-3665e380-37a5-11ea-8c93-ce41f7aee071.png)

imo it would make more sense to open a new PR for it.